crates/ruff_linter/src/rules/flake8_comprehensions/rules/unnecessary_comprehension_any_all.rs

```diff
@@ -29,6 +29,12 @@ use crate::rules::flake8_comprehensions::fixes;
 /// 212 ns ± 0.892 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
 /// ```
 ///
+/// This performance difference is due to short-circuiting; if the entire iterable has to be
+/// traversed, the comprehension version may even be a bit faster (list allocation overhead is not
+/// necessarily greater than generator overhead.)
```

Comment:
Should the period be moved? Always awkward with a parenthetical at the end of a sentence.

```suggestion
/// necessarily greater than generator overhead).
```

Reply:
Oh yeah, good spot; the parenthetical isn't its own sentence, so it should go outside in this case I think.